resources/views/layout.blade.php

```diff
@@ -101,29 +129,41 @@
                                         <li class="dropdown-header">Email Address</li>
                                         <li>{!! link_to_route("mship.manage.email.add", "Add Email") !!}</li>
                                         <li>{!! link_to_route("mship.manage.email.assignments", "SSO Assignments") !!}</li>
-                                    </ul>
-                                </li>
-                                <li class="col-sm-6">
-                                    <ul>
-                                        <li class="dropdown-header">Third-Party Services</li>
-                                        <li>{!! link_to_route("teamspeak.new", "TS Registration") !!}</li>
-                                        <li>{!! link_to_route("slack.new", "Slack Registration") !!}</li>
+                                        @if(Session::has('auth.vatsim-sso'))
+                                            <li class="divider"></li>
+                                            <li>
+                                                <a href="{{ route('logout') }}" onclick="event.preventDefault(); document.getElementById('logout-form').submit();">LOG OUT</a>
+                                            </li>
+                                        @endif
                                     </ul>
                                 </li>
                             </ul>
-
                         </li>
                     </ul>
-                @endif
-                @if(Auth::check() || Session::has('auth.vatsim-sso'))
-                    {!! Form::open(['route' => 'logout', 'id' => 'logout-form']) !!}
-                    <ul class="nav navbar-nav navcustom navbar-right">
-                        <li><a href="{{ route('logout') }}" onclick="event.preventDefault(); document.getElementById('logout-form').submit();">LOG OUT</a></li>
-                    </ul>
                     {!! Form::close() !!}
+
+                    <ul class="nav navbar-nav navcustom navbar-right navbar-notification hidden-xs hidden-sm">
+                        <li class="dropdown dropdown-large">
+                            <a href="{{ route("mship.notification.list") }}" title="Notifications">
+								@if(Auth::user()->has_unread_notifications)
+									<i class="fa fa-bell unread-notifications"></i>
+								@else
+									<i class="fa fa-bell"></i>
+								@endif
+							</a>
+                        </li>
+                    </ul>
+					@if(Auth::user()->roles[0]["id"]==1)
```

Comment:
This should be something like `@if (Auth::user()->hasPermission('adm/dashboard'))`, because most people who have access to the dashboard don't have the role that gives all permissions.